src/platform/site-wide/sass/modules/_m-side-nav.scss

```diff
@@ -15,212 +14,223 @@ $side-nav-color-dark: #323a45;
+    }
 
-  .line-open {
-    background: $side-nav-color-dark;
-    display: block;
-    height: 0.5px;
-    border: 0;
-    margin: 16px 0;
-    padding: 0;
-  }
+    .fa {
+      color: $color-primary;
+    }
 
-  .va-sidenav-toggle-expand {
-    align-items: center;
-    background: $color-gray-lightest;
-    border-radius: 4px;
-    color: $color-primary;
-    display: flex;
-    height: 24px;
-    justify-content: center;
-    width: 24px;
-  }
+    .line {
+      background: $color-gray-lighter;
+      height: 1px;
+      margin: 10px 23px;
+    }
 
-  .va-sidenav-item-label {
-    align-items: center;
-    background: none;
-    border-radius: 0;
-    color: $color-gray-dark;
-    display: flex;
-    font-size: 15px;
-    justify-content: space-between;
-    letter-spacing: 0.3px;
-    line-height: 1.5;
-    margin: 0;
-    padding: 10px 23px;
-    text-align: left;
-    text-decoration: none;
-    text-transform: uppercase;
-    transition: background 0.5s ease;
-    width: 100%;
+    .line-open {
+      background: $side-nav-color-dark;
+      display: block;
+      height: 0.5px;
+      border: 0;
+      margin: 16px 0;
+      padding: 0;
+    }
 
-    &.va-sidenav-item-label-underlined {
-      text-decoration: underline;
+    .va-sidenav-toggle-expand {
+      align-items: center;
+      background: $color-gray-lightest;
+      border-radius: 4px;
+      color: $color-primary;
+      display: flex;
+      height: 24px;
+      justify-content: center;
+      width: 24px;
     }
 
-    &.grandchild-open {
-      margin-left: 10px;
-      border-left: 10px solid #d6d7d9;
+    .va-sidenav-item-label {
+      align-items: center;
+      background: none;
+      border-radius: 0;
+      color: $color-gray-dark;
+      display: flex;
+      font-size: 15px;
+      justify-content: space-between;
+      letter-spacing: 0.3px;
+      line-height: 1.5;
+      margin: 0;
+      padding: 10px 23px;
+      text-align: left;
+      text-decoration: none;
+      text-transform: uppercase;
+      transition: background 0.5s ease;
+      width: 100%;
+
+      &.va-sidenav-item-label-underlined {
+        text-decoration: underline;
+      }
+
+      &.grandchild-open {
+        margin-left: 10px;
+        border-left: 10px solid #d6d7d9;
+      }
     }
-  }
 
-  // Couldn't make use borders efficiently for the different item states
-  // applying borders directly makes the inside content jump and this is not desirable
-  // used backgrounds instead
-  .open {
-    color: $side-nav-color-dark !important;
-    background: linear-gradient($side-nav-color-dark, $side-nav-color-dark) left/ 8px 100%,
-      $side-nav-color-gray-lightest;
-    background-repeat: no-repeat;
-  }
+    // Couldn't make use borders efficiently for the different item states
+    // applying borders directly makes the inside content jump and this is not desirable
+    // used backgrounds instead
+    .open {
+      color: $side-nav-color-dark !important;
```

Comment:
NIT: We should avoid using `!important`, if possible. 

> Using !important, however, is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets. - https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception